tests/__init__.py

```diff
@@ -154,3 +154,13 @@ class SubscriptedList:
 @dataclass
 class SubscriptedDict:
     dict: Dict[str, int] = field(default_factory=lambda: {"foo": 4})
+
+
+@dataclass
+class InterpolationList:
+    list: List[float] = "${optimization.lr}"  # type: ignore
```

Comment:
Check the docs for interpolation with Structured Configs.
you can do something like:
```
list: List[float] = II("optimization.lr")
```
And this would not need the type: ignore.